xvelmap/xr_accessor.py

```diff
@@ -63,6 +66,11 @@ def plot(self, u_var, v_var, lat_dim='latitude', lon_dim='longitude',
                     .format(var_name, (lat_dim, lon_dim), var_dims)
                 )
 
+            # Check if dataset contains nans. If so replace with 0
+            if np.any(xu.isnan(self._ds[var_name]).values):
+                self._ds[var_name] = self._ds[var_name].fillna(0)
```

Comment:
We may not want to modify the dataset in place when plotting.

It would be better to use a copy of `self._ds` or use even simpler data structures (just numpy arrays) for the data that we need to plot (u_var, v_var and lat, lon coordinates).

Reply:
I added `_ds = xr.Dataset()` at the top.